setup.py

```diff
@@ -30,6 +30,8 @@
     packages=find_packages(exclude=["ez_setup", "examples", "tests"]),
     include_package_data=True,
     zip_safe=True,
-    extras_require={"test": ["pytest>=4.6", "pytest-cov", "pydocstyle",
-                             "flake8", "coveralls"]},
+    extras_require={
+        "test": ["pytest>=4.6", "pytest-cov"],
+        "dev": ["pydocstyle", "flake8", "coveralls"],
```

Comment:
`pydocstyle / flake8 / coveralls` are not needed for `tests`. User might want them for `linting / coverage` so I moved them in a `dev` extra